src/planet-a/Active/SysPlanetA.cpp

```diff
@@ -75,30 +74,34 @@ StatusActivated SysPlanetA::activate_sat(
     ActiveEnt root = scene.hier_get_root();
     ActiveEnt planetEnt = scene.hier_create_child(root, "Planet");
 
-    auto &planetTransform = scene.get_registry()
-                            .emplace<osp::active::ACompTransform>(planetEnt);
-    planetTransform.m_transform = Matrix4::translation(positionInScene);
+    auto &rPlanetTransform = scene.get_registry()
+                             .emplace<osp::active::ACompTransform>(planetEnt);
+    rPlanetTransform.m_transform = Matrix4::translation(positionInScene);
     scene.reg_emplace<ACompFloatingOrigin>(planetEnt);
 
-    auto &planetPlanet = scene.reg_emplace<ACompPlanet>(planetEnt);
-    planetPlanet.m_radius = loadMePlanet.m_radius;
+    auto &rPlanetPlanet = scene.reg_emplace<ACompPlanet>(planetEnt);
+    rPlanetPlanet.m_radius = loadMePlanet.m_radius;
+
+    auto &rPlanetForceField = scene.reg_emplace<ACompFFGravity>(planetEnt);
 
-    auto &planetForceField = scene.reg_emplace<ACompFFGravity>(planetEnt);
+    // gravitational constant
+    static const float sc_GravConst = 6.67408E-11f;
```

Comment:
Not 9.8?

Should this go into a globals.h file somewhere? You can also make it a template variable, like such.

template<typename T>
inline constexpr T sgc_gravityConstant = T(6.67408E-11);

And then provide more or less accuracy for specializations.

template<>
inline constexpr float sgc_gravityConstant<float> = 6.67408E-11f;

template<>
inline constexpr double sgc_gravityConstant<double> = // some other number here

Reply:
It's newton's gravity constant 6.67*10^-11, but yes it should go into a global constants file somewhere

Reply:
This is a good idea. I kept it self-contained to stop this PR from getting any larger. The file name should imply something scientific, like "sciencitifc_globals.h," "si_globals.h," or part of the subdirectory name.